profitbricks-rest/src/test/java/org/apache/jclouds/profitbricks/rest/features/VolumeApiMockTest.java

```diff
@@ -102,7 +104,35 @@ public void testCreate() throws InterruptedException {
       
       assertEquals(server.getRequestCount(), 1);
       assertSent(server, "POST", "/rest/datacenters/datacenter-id/volumes", 
-              "{\"properties\": {\"name\": \"jclouds-volume\", \"size\": 3, \"licenceType\": \"LINUX\"}}"
+              "{\"properties\": {\"name\": \"jclouds-volume\", \"size\": 3, \"licenceType\": \"LINUX\", \"type\":\"HDD\"}}"
+      );
+   }
+   
+   @Test
+   public void testCreateWithSsh() throws InterruptedException {
+      server.enqueue(
+         new MockResponse().setBody(stringFromResource("/volume/get.json"))
+      );
+      
+      List<String> sshKeys = new ArrayList<String>();
+      sshKeys.add("hQGOEJeFL91EG3+l9TtRbWNjzhDVHeLuL3NWee6bekA=");
```

Comment:
Is this meant to be the contents of an ssh public key? Is this a real value, or could we use a real one here?